src/backend/gl/src/command.rs

```diff
@@ -624,6 +634,15 @@ impl command::RawCommandBuffer<Backend> for RawCommandBuffer {
     ) {
         unimplemented!()
     }
+    
+    fn push_compute_constants(
```

Comment:
I wonder if we should just make `unimplemented!()` a default implementation on most of the methods (at least, of `Device`). That would save a bit of code in the `empty` device and ease up maintenance during development